src/plugins/console/public/application/containers/editor/legacy/console_editor/editor.tsx

```diff
@@ -255,6 +246,53 @@ function EditorUI({ initialTextValue }: EditorProps) {
             data-test-subj="request-editor"
           />
         </label>
+        <EuiControlBar
+          position="absolute"
+          controls={[
+            {
+              controlType: 'spacer',
+            },
+            {
+              id: 'label',
+              controlType: 'text',
+              text: (
+                <EuiText size="s" color="subdued">
+                  Current Request
+                </EuiText>
+              ),
+            },
+            {
+              controlType: 'text',
+              id: 'copy_to_curl',
+              text: (
+                <EuiToolTip content="Copy to cURL (keyboard shortcut)">
+                  <EuiIcon type="console" color="primary" />
```

Comment:
I'd recommend changing these "primary" colored icons to "ghost" in this situation as the primary color doesn't have a high enough contrast right now. This looks to be a bug with the `EuiControlBar` because if I recall correctly, it should be using the dark mode version of the colors. I should know, though, I built the control bar. 🙈

After looking at the control bar code, I believe that you can stick with the primary color if you turn `EuiIcon` to `EuiButtonIcon`. It would appear that the SCSS lacks styling to properly set a contrast color on just icons. I'll open an issue in EUI about it.